config/intellij-idea-inspections.xml

```diff
@@ -4616,6 +4616,8 @@
     ,com.google.checkstyle.test.base.AbstractModuleTestSupport,verify.*, ,
     ,com.puppycrawl.tools.checkstyle.internal.TestUtil,assert.*, ,
     ,com.puppycrawl.tools.checkstyle.grammar.AstRegressionTest.AssertGeneratedJavaLexer,verify.*, ,
+    ,com.puppycrawl.tools.checkstyle.grammar.AstRegressionTest.AssertGeneratedTextBlockLexer,
```

Comment:
@pbludov I have added this line, which suppresses the warning locally, but TeamCity is still failing.  Any ideas?

Reply:
This may be due to the line break. The IDEA configuration file is intended for machine reading, not for humans. If this is true, rewrite it as a single line and add an exception to check for the line length.

Reply:
TeamCity is still failing...  This is confounding.